pkg/logconfig/config.go

```diff
@@ -29,23 +28,8 @@ const (
 
 	// Controller is the name of the override key used inside of the logging config for Controller.
 	Controller = "controller"
-
-	// Webhook is the name of the override key used inside of the logging config for Webhook Controller.
-	WebhookNameEnv = "WEBHOOK_NAME"
 )
 
 func WebhookName() string {
```

Comment:
I think we can deprecate this function and replace its usages in this repository, and eventually remove it once downstream repositories have migrated.

Reply:
If I remove this downstream tests fails which we can ignore for this case ?

Reply:
By deprecate you mean just add a comment that it's deprecated ?

Reply:
Yes, somethink like: ```DEPRECATED: use knative.dev/pkg/webhook.NameFromEnv()```.

Reply:
Done